packages/ui-components/src/components/calendar-advanced-date-selector/calendar-advanced-date-selector.helper.ts

```diff
@@ -1,30 +1,41 @@
 import { capitalize } from 'lodash-es';
-import { calculateDate, formatTimezoneName, getDefaultTimezone } from '../../utils/date.helper';
+import { calculateDate, formatTimezoneName, getDefaultTimezone, getTimezoneOffset } from '../../utils/date.helper';
 import { SelectedRange } from '../calendar/calendar.types';
 import { ISingleSelectDropdownOption, ISingleSelectDropdownOptions } from '../single-select-dropdown/single-select-dropdown.types';
 import { DEFAULT_TIMEZONE_GROUP_NAME, OTHER_TIMEZONES_GROUP_NAME } from './calendar-advanced-date-selector.config';
-import { RelativeTimeOption } from './calendar-advanced-date-selector.types';
+import { ITimezoneOffset, RelativeTimeOption } from './calendar-advanced-date-selector.types';
 
-const buildDropdownOption = (timezone: string, group: string = OTHER_TIMEZONES_GROUP_NAME): ISingleSelectDropdownOption => ({
-	label: formatTimezoneName(timezone),
+const buildDropdownOption = (label: string, timezone: string, group: string = OTHER_TIMEZONES_GROUP_NAME): ISingleSelectDropdownOption => ({
 	value: timezone,
+	label,
 	group
 });
 
-export const buildTimezonesDropdownOptions = (timezones: string[]): ISingleSelectDropdownOptions => {
-	const defaultTimezone = getDefaultTimezone();
+export const buildTimezoneByOffset = (timezones: string[]): ITimezoneOffset[] => {
+	return timezones
+		.map(name => {
+			const offset = getTimezoneOffset(name);
+			const label = formatTimezoneName(name);
+
+			return { offset, name, label };
+		})
+		.sort((name1, name2) => name1.offset - name2.offset);
```

Comment:
You can sort by name, right? 
Since the offset is on the suffix, sorting by name will give exactly the same output and also sort alphabetically by the timezone name.

Reply:
it's not possible. The localCompare don't use correctly the negative values.

Reply:
That makes sense, sorry 👍